src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -32,5 +32,16 @@ internal void InitializeDefaultAutoCompleteCandidates()
             }
             FilteredResults = candidates;
         }
+
+        internal void PopulateAutoCompleteCandidates()
+        {
+            var searchElements = PortViewModel.GetMatchingNodes();
```

Comment:
Thanks for moving it. But I was hoping we can move all of the code from `PortViewModel` to the customized search view model here given that 1. those code are not really related to PortView 2. I feel `PortViewModel` does not necessarily obtain knowledge about search elements 3. We still have access to DynamoViewModel from here. What do you think?

Reply:
I haven't moved the code yet. Doing so now.